tests/test_commands.py

```diff
@@ -195,6 +195,39 @@ def teardown():
         users = r.acl_list()
         assert 'user %s off -@all' % username in users
 
+    @skip_if_server_version_lt(REDIS_6_VERSION)
+    def test_acl_log(self, r, request):
+        username = 'redis-py-user'
+
+        def teardown():
+            r.acl_deluser(username)
+
+        request.addfinalizer(teardown)
+        r.acl_setuser(username, enabled=True, reset=True,
+                      commands=['+get', '+set', '+select'],
+                      keys=['cache:*'], nopass=True)
+        r.acl_log_reset()
+
+        r_test = redis.Redis(host='localhost', port=6379, db=9,
```

Comment:
We don't want to hardcode connection info in a test. Instead, use the `_get_client` function from `conftest.py`. Something like:

```python
r_test = _get_client(redis.Redis, request, username=username)
```

Reply:
Yep please check the comment below